docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -97,164 +83,204 @@ can be seen in the Information tab, as shown in the image below. For more inform
 to :ref:`control_center_tutorial`. 
 
 .. _hobby_fc_tutorial_motor_configuration:
 
-Motor Configuration
-===================
-.. warning:: Before setting any parameters, ensure that the motor does NOT have a propeller attached and
-     is held in place securely. If the motor is unsecured it could move unexpectedly when it starts to spin. 
-     This could damage the motor, its connectors, or any nearby observers.
+Module Configuration
+=====================
+.. warning:: Before setting any parameters, ensure that the module does NOT have a propeller attached and
+     is held in place securely. If the module is unsecured it could move unexpectedly when it starts to spin. 
+     This could damage the module, its connectors, or any nearby observers.
 
-Now that the motor is setup, we can begin configuring it with Control Center. The General tab and the Tuning tab are the only ones with parameters that need to be configured.
+Now that the module is setup, you can begin configuring it with Control Center. The General tab and the Tuning tab are the only tabs with parameters that need to be configured.
 If you are unfamiliar with setting parameters through the Control Center, refer to :ref:`control_center_tutorial`.
 
 General Tab
 ***********
 First, there are a few parameters to set in the General tab. Not all of the parameters in this tab are important, and only the relevant ones are
 highlighted here. If a parameter is not mentioned, you can safely leave it at its default. The correct settings and purpose of each parameters are 
-described in greater detail below. The figure below provides a quick overview of what the settings in Control Center should look like when you're done setting up for PWM, with
-the important parameters indicated with a red dot. Note that the figure is from an 8108, you may have less total parameters in the General tab on a 2306, but the 
-important parameters should be the same between both sizes.
+described in detail in the sections below. **Most parameters are the same in Standard PWM and DSHOT setups, but ones that differ have sections for both setups**:
+
+Communication
+##############
+This parameter controls what type of :ref:`Hobby Protocols <manual_hobby>` the module will listen for. The module is capable of listening to a wide variety of hobby protocols,
+which can be selected from the drop-down box. Once the module receives a command in its selected hobby protocol, it will only listen for that type of command until power cycled, so you will
+not be able to connect with Control Center until you power cycle the module. For example, if the module receives a PWM message when it is set to accept them, it will only listen to PWM messages until it is reset. 
+If left in Autodetect mode, the module should listen for all of the supported Hobby protocols to try to determine which one you are using. The module can sometimes have trouble automatically discovering the protocol though 
```

Comment:
The module may have difficulty automatically discovering the protocol, however, and is affected by the flight controller as well as noise on the communication lines.